pages/common/peerflix.md

```diff
@@ -1,15 +1,23 @@
 # peerflix
 
-> Stream video or audio based torrents to your favorite player.
+> Stream video- or audio-based torrents to a media player.
 
-- Stream a torrent:
+- Stream the largest media file in a torrent given as a magnet link:
```

Comment:
Ah so its the largest file ! Great, I had this doubt earlier. Sorry it got slipped.

Reply:
No problem. I was feeling uneasy about the length of the discussion in the previous PR so I didn't want to introduce more friction there, since we could always introduce further changes in separate PRs. After all, gradual improvement is what collaborative projects are all about :)